data/data-pipeline/data_pipeline/tests/sources/national_risk_index/test_etl.py

```diff
@@ -151,13 +126,27 @@ def test_extract(self, mock_etl, mock_paths):
         extracted_file_path = (
             tmp_path / "NationalRiskIndexETL" / "NRI_Table_CensusTracts.csv"
         )
+
         assert extracted_file_path.is_file()
 
-        input_csv_path = (
-            self._DATA_DIRECTORY_FOR_TEST / self._INPUT_CSV_FILE_NAME
-        )
+    def test_extract(self, snapshot, mock_etl, mock_paths):
+        """Tests the extract method. I think this works
 
-        # Make sure extracted file is equal to the input fixture:
-        assert filecmp.cmp(
-            f1=extracted_file_path, f2=input_csv_path, shallow=False
+        To update: $ poetry run pytest
+            data_pipeline/tests/sources/national_risk_index/test_etl.py::test
+            --snapshot-update
+        """
```

Comment:
Appreciate the guidance here on how to update the test fixtures for _just_ this ETL class

Reply:
I moved this into the header for the class